enterprise/backend/test/metabase_enterprise/sso/integrations/jwt_test.clj

```diff
@@ -202,6 +202,9 @@
                        :common_name  "New User"}]
                      (->> (mt/boolean-ids-and-timestamps (t2/select User :email "newuser@metabase.com"))
                           (map #(dissoc % :last_login))))))
+            (testing "User Invite Event is logged."
+              (is (= "newuser@metabase.com"
+                     (get-in (t2/select-one :model/AuditLog :topic :user-invited) [:details :email]))))
```

Comment:
added this inside the test to piggyback on the setup/teardown so that it's not doubled up for this single check.